jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandler.java

```diff
@@ -720,7 +685,9 @@ public boolean isProgrammaticListener(EventListener listener)
     
     public boolean isDurableListener(EventListener listener)
     {
-        return _durableListeners.contains(listener);
+        if (isStarted())
```

Comment:
Can you explain the logic behind this change? Durable listeners have a very specific meaning. I'm not sure its correct to be returning listeners that are not specifically in the durable list.

Reply:
Durable listeners are copied when the context is started, so before start all listeners are durable.
Adding comments to say that.